modules/electromagnetics/doc/content/source/auxkernels/AzimuthMagneticTimeDerivScalar.md

```diff
@@ -0,0 +1,28 @@
+# AzimuthMagneticTimeDerivScalar
+
+!syntax description /AuxKernels/AzimuthMagneticTimeDerivScalar
+
+## Overview
+
+!style halign=left
+The AzimuthMagneticTimeDerivScalar object calculates the time derivative of the azimuthal magnetic field for an axisymmetric cylindrical configuration. The term is define as:
```

Comment:
does this object only work in RZ coordinates then? we could add RZ to the name if so?

Reply:
Yes, the derivation is for RZ coordinates and I can add RZ at the end.